src/Shared/LayoutRenderers/AspNetResponseHeadersWrittenLayoutRenderer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET response headers already sent
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-response-headers-written}
+    /// </remarks>
+    [LayoutRenderer("aspnet-response-headers-written")]
+    public class AspNetResponseHeadersWrittenLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Could this be renamed to `AspNetResponseHasStartedLayoutRenderer` ?

Reply:
Good idea, we should use the ASP.NET Core name, not the legacy Framework name